src/main/java/chronologer/parser/DateTimeExtractor.java

```diff
@@ -47,6 +47,9 @@ public static LocalDateTime extractDateTime(String dateTimeFromUser, String comm
         } else if (command.equals("view")) {
             dateView = LocalDateTime.parse(dateTimeFromUser, DATE_FORMATTER);
             return dateView;
+        } else if (command.equals("assignment")) {
+            dateView = LocalDateTime.parse(dateTimeFromUser, DATE_FORMATTER);
+            return dateView;
```

Comment:
is it just me or the DateTimeExtractor is too coupled to our parsers? We have to constantly modify it if we add new commands that takes in dates

Reply:
We don't even need any of the command checks in the first place since we are using the same Date formatter regardless. We can simplify this entire extractor to <6 ish lines of code.